tests/Predis/Connection/Replication/SentinelReplicationTest.php

```diff
@@ -51,17 +51,17 @@ public function testParametersForSentinelConnectionShouldUsePasswordForAuthentic
     /**
      * @group disconnected
      */
-    public function testParametersForSentinelConnectionShouldIgnoreDatabaseAndPassword(): void
+    public function testParametersForSentinelConnectionShouldIgnoreDatabase(): void
```

Comment:
I think we should merge this into `main` instead, unless we can keep the existing unit tests unchanged.

Reply:
Well, functionality was changed and unit test should be changed according to this. Or you mean that the whole PR should be merged into main?

Reply:
What I mean is that people might rely on the removed/changed functionality and if we want to change behavior, we have to either merge into `main` because it's breaking.

I'm usually being overly precautious and assume people have weird Sentinel setups with varying usersnames for each node, or some other oddities.

If you like we can merge this into `2.x`, but we might crash someone's setup and need to revert.

Reply:
You're right, it's a breaking change. Here's new PR against main #1429